packages/cpp/src/browser/cpp-build-configurations.ts

```diff
@@ -131,9 +131,10 @@ export class CppBuildConfigurationChanger implements QuickOpenModel {
     async onType(lookFor: string, acceptor: (items: QuickOpenItem[]) => void): Promise<void> {
         const items: QuickOpenItem[] = [];
         const active: CppBuildConfiguration | undefined = this.cppBuildConfigurations.getActiveConfig();
+        const configurations = Array.from(this.cppBuildConfigurations.getConfigs()).sort();
 
         // Add one item per build config.
-        this.cppBuildConfigurations.getConfigs().forEach(config => {
+        configurations.forEach(config => {
```

Comment:
These changes are probably not needed anymore?

Reply:
If you prefer, we can remove the ``sort()`` function on the array of configs, else we can clean the code to a single line as the following:
 ``Array.from(this.cppBuildConfigurations.getConfigs()).sort().forEach(config => ... ``

Reply:
Sorry, I had not noticed that you had left the sort.  Nevermind.